tests/main/mounts-persist-refresh-content-snap/task.yaml

```diff
@@ -55,9 +56,10 @@ execute: |
   "$TESTSTOOLS"/snaps-state install-local test-snapd-content-slot
 
   # Signal to kill the loop
-  rm ~test/snap/test-snapd-desktop-layout-with-content/common/keep-running
+  rm -f ~test/snap/test-snapd-desktop-layout-with-content/common/keep-running
   wait "$pid"
 
   # Ensure that /usr/share/fonts/foo-font was never missing.
-  MATCH 'exited' ~test/snap/test-snapd-desktop-layout-with-content/common/status
-  NOMATCH 'foo-font missing' ~test/snap/test-snapd-desktop-layout-with-content/common/status
+  MATCH 'exited' <~test/snap/test-snapd-desktop-layout-with-content/common/status
```

Comment:
Is the purpose of this check just to check that something didn't go drastically wrong and ensure that crash-foo-font ran at least once?

Reply:
Yeah, we want to make sure we're not watching partial content.